app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [{"line": ers["line_number"],
+                    "column": ers["column_number"],
+                    "message": ers["text"], "name": ers["code"],
+                    "source": "flake8"} for ers in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [{"errors": [{"line": e["line_number"],
```

Comment:
use format_single_linter_file function to avoid code duplication

Reply:
Do you see my new commit?